vj4/server.py

```diff
@@ -10,7 +10,7 @@
 from vj4 import app
 from vj4.util import options
 
-options.define('listen', default='http://127.0.0.1:8888', help='Server listening address.')
```

Comment:
This will cause a security problem.
I think most servers (no matter web server or not) default listen to localhost.